lib/circuitry/event_queue_creator.rb

```diff
@@ -0,0 +1,86 @@
+module Circuitry
+  class EventQueueCreator
+
+    attr_reader :sqs, :sns, :queue_name
+
+
+    def initialize(queue_name, sqs: Aws::SQS::Client.new, sns: Aws::SNS::Client.new)
+      @queue_name = queue_name
+      @sqs, @sns = sqs, sns
+    end
+
+    def verify_queue(topics)
```

Comment:
Yeah, I agree on the word choice. I've struggled with this in the past, but don't have a good answer. I've come down to using `validate` to just inspect and report errors, and `verify` to inspect and correct errors. This method doesn't do it yet, but I envisioned it to modify queue settings (like visibility timeout) if needed.